package.json

```diff
@@ -73,7 +73,9 @@
     "karma-webpack": "latest",
     "lodash": "latest",
     "mocha": "latest",
-    "msgpack-test-js": "latest",
+    "msg-ext": "^1.0.1",
+    "msg-int64": "^0.1.1",
+    "msg-timestamp": "^1.0.1",
```

Comment:
These 3 were deep dependencies of `msgpack-test-js`, the rest of which I inlined into this project (see the new `test/utils` folder).

Reply:
Can you explain why you had to inline `msgpack-test-js`? Was something broken that you had to fix -- if so, what specifically did you change?

Reply:
Yeah, the core of it was JSON.stringify'ing an object that had BigInt in it and dying

Reply:
Plus frankly, it was weird that stuff was external anyway, it's code that clearly should live in this project.